hydra-node/src/Hydra/Logging.hs

```diff
@@ -77,7 +77,9 @@ withTracer (Verbose name) action = do
         , "message" .= msg
         ]
 
-  logMessagesToStdout queue = forever $ flushLogs queue
+  logMessagesToStdout queue =
+    forever $
+      flushLogs queue >> threadDelay 0.1
 
   flushLogs queue = do
     entries <- atomically $ flushTBQueue queue
```

Comment:
I misread `flushTBQueue`'s documentation, it says "This function never retries" which obviously means it alwasy returns and never blocks. I am not a big fan of using `threadDelay`, perhaps we should instead use `readTBQueue` in conjunction with `lengthTBQueue` to make sure we block if there's nothing to read, eg. something like
```
do
   len <- lengthTBQueue q
   check $ len > 0
   replicateM (readTBQueue q)
```
instead?

Reply:
Why not just `forever $ readTBQueue >>= putBSLn . encode`?

Reply:
Plus the stdout flushing